contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/SpecialPermissionsList.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div data-test="special-permissions-list">
+    <div class="header-section">
+      <div class="title">
+        {{ specialPermissionsDetected$() }}
+      </div>
+      <div class="description">
+        {{ confirmDistributionRights$() }}
+      </div>
+    </div>
+
+    <div
+      v-if="isLoading"
+      class="loader-wrapper"
+    >
+      <KCircularLoader />
+    </div>
+
+    <template v-else-if="currentPagePermissions.length > 0">
+      <div class="permissions-box">
+        <KCheckbox
+          v-for="permission in currentPagePermissions"
+          :key="permission.id"
+          :checked="checkedIds.includes(permission.id)"
+          :label="permission.description"
+          class="permission-checkbox"
+          @change="togglePermission(permission.id)"
+        />
+      </div>
+
+      <div
+        v-if="totalPages > 1"
+        class="pagination"
+      >
+        <KButton
+          :disabled="currentPage === 1"
+          appearance="basic-link"
+          class="nav-button"
+          @click="previousPage"
+        >
+          &lt; Previous
```

Comment:
Seems like we have some unwrapped strings around here 👀. Also, we can use the [KLabeledIcon](https://design-system.learningequality.org/klabeledicon) component, with this [icon](https://design-system.learningequality.org/icons#token-chevronLeft)